std/conv.d

```diff
@@ -2173,9 +2174,11 @@ This overload converts a character input range to a `bool`.
 Params:
     Target = the type to convert to
     source = the lvalue of an $(REF_ALTTEXT input range, isInputRange, std,range,primitives)
+    doCount = the flag for deciding to report the number of consumed characters
 
 Returns:
-    A `bool`
+    A `bool` if doCount is set to No.doCount
```

Comment:
Please wrap function arguments (`doCount`, `No.doCount`) within backticks ( **\`** ), as you do with `bool`

Reply:
Please make this an itemized unordered list.
The syntax is:
```
/** 
  * $(UL
  *   $(LI Item 1)
  *   $(LI Item 2))
  */
```